docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2060,8 +2060,13 @@ Result:
 
 ### getRecentPerformanceSamples
 
-Returns a list of recent performance samples, in reverse slot order. Performance samples are taken every 60 seconds and
-include the number of transactions and slots that occur in a given time window.
+Returns a list of recent performance samples, in reverse slot order. Performance
+samples are taken every 60 seconds and include the number of transactions and
+slots that occur in a given time window.
+
+`numNonVoteTransaction` is the number of non-voting transactions in the sample.
+To get the number of voting transactions, compute `numTransactions -
+numNonVoteTransaction`.
```

Comment:
nit: To remain consistent with the style of the rest of this page / avoid some redundancy, I think this note could follow `numNonVoteTransaction` below under the `Results`.

Reply:
Moved and rephrased to match the context.